app.config.ts

```diff
@@ -33,19 +33,26 @@ export default ({ config }: ConfigContext): ExpoConfig => ({
   },
   version: appBuildNumbers.expo.version,
   assetBundlePatterns: ["**/*"],
+  plugins: [
+    "@react-native-firebase/app-check",
```

Comment:
I believe with https://github.com/ephemeraHQ/converse-app/pull/1356, we'll need this for Android, but not for iOS since it's in a bare expo workflow. cc @nmalzieu @alexrisch (Kyle had this here IIRC before I assumed the work)

Reply:
And they still won't be used in iOS a little bit confusingly, but that's okay for now. iOS would be much more effort to get to be using Expo Continuous Native Generation with our notification extension. 

But I definitely think looking into it would be worth our time eventually.